README.md

```diff
@@ -1,70 +1,157 @@
+<details>
+<summary>Components</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Components</a></li>
+  </ul>
+</details>
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<!-- Features -->
 
-## Learn More
+### Key Features <a name="key-features"></a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
-To learn React, check out the [React documentation](https://reactjs.org/).
 
-### Code Splitting
+- **[Redux]**
+- **[React]**
+- **[Javascript]**
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Analyzing the Bundle Size
+<!-- GETTING STARTED -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Making a Progressive Web App
+Git clone https://github.com/SAZZAD1Q2/GlobeTrekker.git
+Run in your browser
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Prerequisites
 
-### Advanced Configuration
+In order to run this project you need:
+Git
+An IDE like vs code . bracket etc.
+Code Editor
+Browser
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+### Setup
 
-### Deployment
+Clone this repository to your desired folder: https://github.com/SAZZAD1Q2/GlobeTrekker.git
+Opent it to your code editor
+open it then to your browser.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
 
-### `npm run build` fails to minify
+### Install
+
+Install this project with:
+Github or git or vs code. For this open your terminal and proceed some steps:
+a. Install node package using the command : npm install
+b. Then run the website using the command : npm start
+c. Test the code using jest snapShots and for this use the Command : npx jest
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### deployment
+
+You may use the online version of my work. Here is the link of online version: [Link](https://globe-trekker.vercel.app/)
```

Comment:
- [ ] Please add the correct deployed link now your link is not showing your project check below my video for more clarification :slightly_smiling_face: 

_**Note: Please watch this video on Chrome browser because this video is only played on Chrome browser**_

[screen-capture (20).webm](https://github.com/SAZZAD1Q2/GlobeTrekker/assets/45226848/38e86d22-33c7-4f7e-ad8c-675d6efcd77e)